api/market.ts

```diff
@@ -89,3 +90,31 @@ export async function fetchDepositApyByEpochsDeposited(
       }
     })
 }
+
+export async function fetchEpochs() {
+  const currentTimestamp = Math.floor(Date.now() / 1000)
+  const markets = await fetchMarkets()
+  const marketsByOneAsset = markets
+    .filter(
+      (market) =>
+        market.endTimestamp > currentTimestamp &&
+        isAddressEqual(
+          market.quoteToken.address,
+          markets[0].quoteToken.address,
```

Comment:
I think it might be better to get the address as an argument in case there are markets with less epochs.

For example,

USDC might have 3 epochs, while WETH has 4 epochs.